R/declare_statistical_model.R

```diff
@@ -0,0 +1,27 @@
+
+#' @export
+declare_statistical_model <- make_declarations(statistical_model_handler, step_type = "wallet", causal_type = "estimator")
+
+#' @importFrom rlang eval_tidy quos as_label
+#' @export
+statistical_model_handler <- function(data, ...) {
+  
+  dots <- quos(...)
+  
+  if(length(dots) > 1) {
+    stop("Please only send one model to declare_statistical_model.")
```

Comment:
May as well just loop over the dots and evaluate all of them, instead of stopping here.